src/uct/api/v2/uct_v2.h

```diff
@@ -1050,6 +1050,83 @@ int uct_iface_is_reachable_v2(uct_iface_h iface,
                               const uct_iface_is_reachable_params_t *params);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Parameters passed to @ref uct_md_query_tl_resources_v2.
+ */
+typedef struct uct_md_query_tl_resources_params {
+    /**
+     * Mask of valid fields which must currently be set to zero.
+     * Future fields not specified in this mask will be ignored.
+     * Provides ABI compatibility with respect to adding new fields.
+     */
+    uint64_t                       field_mask;
```

Comment:
This file contains different styles of alignment of the fields of structures. So, I suggest to use the same style in this PR.
```suggestion
    uint64_t field_mask;
```

Reply:
fixed